### PR TITLE
Extract viz.json loading from vis/vis.js to createVis

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
This PR completes the second task of #804.

The create-vis function now fetches the viz.json file from a URL when given one.

@xavijam can you please review? Thanks!

### CODE REVIEW

**Comment by alonsogarciapablo** [comment]
retest this please

**Comment by alonsogarciapablo** [comment]
bump!

File: src/api/create-vis.js [hunk 1]
```diff
@@ -1,34 +1,128 @@
...
+var loadVizJSON = function (vis, vizjsonData, options) {
+  var vizjson = new VizJSON(vizjsonData);
+  applyOptionsToVizJSON(vizjson, options);
+  vis.load(vizjson);
+  if (!options.skipMapInstantiation) {
+    vis.instantiateMap();
+  }
+};
+
+var applyOptionsToVizJSON = function (vizjson, options) {
```
Comment (xavijam) [inline src/api/create-vis.js]: :+1:

**Comment by xavijam** [comment]
LGTM man! :clap: :clap:

**Comment by alonsogarciapablo** [comment]
Thanks!

**Pull request merged**